reference/api/documents.md

```diff
@@ -138,6 +138,8 @@ The body is composed of a **JSON array** of documents.
 
 This `updateId` allows you to [track the current update](/reference/api/updates.md).
 
+[You can set the primary key of the index on document addition.](/learn/core_concepts/documents.md#setting-the-primary-key-on-document-addition)
```

Comment:
This seems a bit pasted here without context (Same for others). It also is redundant with the sentence above stating

> If you want to set the **primary key** of your index through this route, it only has to be done **the first time you add documents** to the index. After which it will be ignored if given.

You can add a link in the sentence above maybe: 

```
If you want to [set the **primary key** of your index](/learn/core_concepts/documents.md#setting-the-primary-key-on-document-addition) ...

```